ballista/rust/scheduler/src/state/task_manager.rs

```diff
@@ -43,17 +43,17 @@ use rand::{thread_rng, Rng};
 use std::collections::HashMap;
 use std::default::Default;
 use std::sync::Arc;
+use std::time::Duration;
 use tokio::sync::RwLock;
-use tonic::transport::Channel;
 
-type ExecutorClients = Arc<RwLock<HashMap<String, ExecutorGrpcClient<Channel>>>>;
 type ExecutionGraphCache = Arc<RwLock<HashMap<String, Arc<RwLock<ExecutionGraph>>>>>;
 
+const CLEANUP_FINISHED_JOB_DELAY_SECS: u64 = 300;
```

Comment:
We should make this configurable. Some of the queries I am testing take much longer than 300 seconds. We already have the ability to set configs on the context.

Reply:
Never mind, this is a delay _after_ the job completes. I would still like to see this configurable but we could do that as a follow in PR.